config.yaml

```diff
@@ -1 +1,11 @@
-dynamo-endpoint: dynamodb.us-east-1.amazonaws.com
\ No newline at end of file
+dynamo-endpoint: http://localhost:8000
+
+# Authentication Credentials
+application: thunder
+secret: secret
```

Comment:
I think you can do lists in the config something like this (you may have to play around with it a bit or Google it, I'm not positive this is the syntax):

```yaml
approvedKeys:
  - application: app
    secret: sec
  - application: app2
    secret: sec2
```